pkg/operator/ceph/controller/spec_test.go

```diff
@@ -605,7 +605,7 @@ func TestGetDaemonsToSkipReconcile(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			namespace := "rook-ceph"
-			daemonName := "nfs"
+			daemonName := "instance"
```

Comment:
`daemonName := "instance"` does not sound right.  Directly use the `instanceLabelKey` in the method invocation.